src/cursor/cur_std.c

```diff
@@ -582,6 +595,100 @@ err:	API_END(session, ret);
 }
 
 /*
+ * __cursor_modify --
+ *	WT_CURSOR->modify default implementation.
+ */
+static int
+__cursor_modify(WT_CURSOR *cursor, WT_MODIFY *entries, int nentries)
```

Comment:
I chose `int` because we don't need a fixed-size value, Windows won't let us use `u_int` in APIs, and `int` matches the POSIX APIs `preadv` and `pwritev`.

Not compelling reasons, but I couldn't think of a reason not to use `int`. Why do you prefer something else?